INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1142 ==================================================
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1143 Starting new test on CHROME
DEBUG    selenium.webdriver.common.service:service.py:215 Started executable: `/usr/local/bin/chromedriver` in a child process with pid: 6077
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session {"capabilities": {"firstMatch": [{}], "alwaysMatch": {"browserName": "chrome", "pageLoadStrategy": "normal", "goog:loggingPrefs": {"performance": "ALL"}, "goog:chromeOptions": {"prefs": {"download.default_directory": "/home/runner/work/selenium-demoblaze-operations/selenium-demoblaze-operations/downloads", "download.prompt_for_download": false, "download.directory_upgrade": true, "safebrowsing.enabled": false, "profile.default_content_settings.popups": 0, "profile.content_settings.exceptions.automatic_downloads.*.setting": 1}, "excludeSwitches": ["enable-logging", "enable-automation"], "useAutomationExtension": false, "extensions": [], "args": ["--headless=new", "--window-size=1920,1080", "--start-maximized", "--disable-gpu", "--no-sandbox", "--disable-dev-shm-usage", "--disable-extensions", "--disable-popup-blocking", "--ignore-certificate-errors", "--disable-web-security", "--allow-running-insecure-content"]}}}}
DEBUG    urllib3.connectionpool:connectionpool.py:244 Starting new HTTP connection (1): localhost:54551
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":{"capabilities":{"acceptInsecureCerts":false,"browserName":"chrome","browserVersion":"141.0.7390.122","chrome":{"chromedriverVersion":"141.0.7390.122 (b477534e7e10d193e916cd4e2967c589383625b2-refs/branch-heads/7390@{#2667})","userDataDir":"/tmp/.org.chromium.Chromium.CEs85U"},"fedcm:accounts":true,"goog:chromeOptions":{"debuggerAddress":"localhost:45791"},"networkConnectionEnabled":false,"pageLoadStrategy":"normal","platformName":"linux","proxy":{},"setWindowRect":true,"strictFileInteractability":false,"timeouts":{"implicit":0,"pageLoad":300000,"script":30000},"unhandledPromptBehavior":"dismiss and notify","webauthn:extension:credBlob":true,"webauthn:extension:largeBlob":true,"webauthn:extension:minPinLength":true,"webauthn:extension:prf":true,"webauthn:virtualAuthenticators":true},"sessionId":"5d123d413bbc08f8641ccb0eb6cc9583"}} | headers=HTTPHeaderDict({'Content-Length': '850', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583/timeouts {"implicit": 10000}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session/5d123d413bbc08f8641ccb0eb6cc9583/timeouts HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583/timeouts {"pageLoad": 60000}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session/5d123d413bbc08f8641ccb0eb6cc9583/timeouts HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.address`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.address` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.automotive`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.automotive` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.bank`.
DEBUG    faker.factory:factory.py:86 Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.barcode`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.barcode` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.color`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.color` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.company`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.company` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.credit_card`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.credit_card` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.currency`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.currency` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.date_time`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.date_time` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.emoji` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.geo`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.geo` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.internet`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.internet` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.job`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.job` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.lorem`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.lorem` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.misc`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.misc` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.passport`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.passport` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.person`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.person` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.phone_number`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.phone_number` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.sbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.ssn`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.ssn` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583/url {"url": "https://www.demoblaze.com"}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session/5d123d413bbc08f8641ccb0eb6cc9583/url HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1163 Navigated to DemoBlaze website on chrome
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1736 Testing performance metrics
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583/url {"url": "https://www.demoblaze.com"}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session/5d123d413bbc08f8641ccb0eb6cc9583/url HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583/execute/sync {"script": "return window.performance.getEntriesByType('navigation')[0];", "args": []}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session/5d123d413bbc08f8641ccb0eb6cc9583/execute/sync HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":{"activationStart":0,"connectEnd":0.5999999999767169,"connectStart":0.5999999999767169,"criticalCHRestart":0,"decodedBodySize":17232,"deliveryType":"cache","domComplete":92.39999999996508,"domContentLoadedEventEnd":89.29999999998836,"domContentLoadedEventStart":89.20000000001164,"domInteractive":88.79999999998836,"domainLookupEnd":0.5999999999767169,"domainLookupStart":0.5999999999767169,"duration":93.09999999997672,"encodedBodySize":4423,"entryType":"navigation","fetchStart":0.5999999999767169,"finalResponseHeadersStart":0,"firstInterimResponseStart":1.8999999999650754,"initiatorType":"navigation","loadEventEnd":93.09999999997672,"loadEventStart":92.39999999996508,"name":"https://www.demoblaze.com/","nextHopProtocol":"h2","notRestoredReasons":null,"redirectCount":0,"redirectEnd":0,"redirectStart":0,"renderBlockingStatus":"non-blocking","requestStart":1.7000000000116415,"responseEnd":5.5,"responseStart":1.8999999999650754,"responseStatus":200,"secureConnectionStart":0.5999999999767169,"serverTiming":[],"startTime":0,"toJSON":{},"transferSize":0,"type":"navigate","unloadEventEnd":12.5,"unloadEventStart":12.5,"workerCacheLookupStart":0,"workerFinalSourceType":"","workerMatchedSourceType":"","workerRouterEvaluationStart":0,"workerStart":0}} | headers=HTTPHeaderDict({'Content-Length': '1266', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1744 Page load time: 92.5 ms
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583/se/log {"type": "browser"}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "POST /session/5d123d413bbc08f8641ccb0eb6cc9583/se/log HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":[{"level":"WARNING","message":"https://www.demoblaze.com/node_modules/video.js/dist/video.min.js 11:694 \"VIDEOJS:\" \"WARN:\" \"A plugin named \\\"reloadSourceOnError\\\" already exists. You may want to avoid re-registering plugins!\"","source":"console-api","timestamp":1761248206242},{"level":"WARNING","message":"https://www.demoblaze.com/node_modules/video.js/dist/video.min.js 11:694 \"VIDEOJS:\" \"WARN:\" \"A plugin named \\\"reloadSourceOnError\\\" already exists. You may want to avoid re-registering plugins!\"","source":"console-api","timestamp":1761248206387}]} | headers=HTTPHeaderDict({'Content-Length': '583', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1747 Browser logs count: 2
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1749 Performance metrics collected
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1168 Test completed on chrome
INFO     TestDemoBlazeE2E_chrome:test_demoblaze_e2e.py:1169 ==================================================
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 DELETE http://localhost:54551/session/5d123d413bbc08f8641ccb0eb6cc9583 {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:54551 "DELETE /session/5d123d413bbc08f8641ccb0eb6cc9583 HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request